src/Features/Core/Portable/Shared/Options/ServiceFeatureOnOffOptions.cs

```diff
@@ -35,5 +35,34 @@ public static bool IsClosedFileDiagnosticsEnabled(OptionSet options, string lang
 
             return option.Value;
         }
+
+        /// <summary>
+        /// Option to disable analyzer execution during live analysis.
+        /// </summary>
+        public static readonly Option<bool> DisableAnalyzers = new Option<bool>(
+            nameof(ServiceFeatureOnOffOptions), nameof(DisableAnalyzers), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation($"Options.DisableAnalyzers"));
+
+        /// <summary>
+        /// Option to turn off all background analysis to improve performance.
```

Comment:
❕ Power save mode is not designed to improve performance, and is not limited to background analysis. This comment should be rewritten to correctly state the intent of the feature, e.g.:

> Option to disable implicit background operations to reduce CPU power consumption.